commit f1d0fe0abc2c7643dee6df5ed473d3f1b0e15a97
Author: Greg Kroah-Hartman <gregkh@suse.de>
Date:   Fri Oct 8 11:13:11 2010 -0700

    Staging: brcm80211: remove unused typedefs in typedefs.h
    
    uint is already defined somewhere else, so just remove this version
    of it.
    
    Cc: Brett Rudley <brudley@broadcom.com>
    Cc: Henry Ptasinski <henryp@broadcom.com>
    Cc: Nohee Ko <noheek@broadcom.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

--- a/drivers/staging/brcm80211/include/typedefs.h
+++ b/drivers/staging/brcm80211/include/typedefs.h
@@ -19,34 +19,16 @@
 
 #include <linux/types.h>
 
-/*
- * Infer the compile environment based on preprocessor symbols and pragmas.
- * Override type definitions as needed, and include configuration-dependent
- * header files to define types.
- */
-
 #if defined(__x86_64__)
 #define TYPEDEF_UINTPTR
 typedef unsigned long long int uintptr;
 #endif
 
-#define TYPEDEF_UINT
-
-/*
- * Default Typedefs
- */
-
-/* define uint */
-
-#ifndef TYPEDEF_UINT
-typedef unsigned int uint;
-#endif
-
 #ifndef TYPEDEF_UINTPTR
 typedef unsigned int uintptr;
 #endif
 
-/* define macro values */
+#undef TYPEDEF_UINTPTR
 
 #ifndef FALSE
 #define FALSE	0
@@ -66,8 +48,6 @@ typedef unsigned int uintptr;
 
 #define	AUTO	(-1)		/* Auto = -1 */
 
-#undef TYPEDEF_UINT
-#undef TYPEDEF_UINTPTR
 
 /*
  * Including the bcmdefs.h here, to make sure everyone including typedefs.h
